src/modules/dynamic-ui-wrapper/dynamic-ui-wrapper.ts

```diff
@@ -120,6 +137,32 @@ export class DynamicUiWrapper {
     this.currentUserTask = undefined;
   }
 
+  private _finishManualTask(): void {
+    const hasNoCurrentManualTask: boolean = this.currentManualTask === undefined;
+
+    if (hasNoCurrentManualTask) {
```

Comment:
(...) this becomes a sentence:

```suggestion
    if (noManualTaskKnown) {
```